ubuntu_example.sh

```diff
@@ -14,7 +14,7 @@ sudo docker run hello-world # Should run!
 # Install RVM
 command curl -sSL https://rvm.io/mpapis.asc | gpg --import -
 curl -sSL https://get.rvm.io | bash
-source /usr/local/rvm/scripts/rvm
+source ~/.rvm/scripts/rvm
```

Comment:
@nicmel Did you get this error on Ubuntu 17.10? And if so, what was the error? I was not able to reproduce this issue yesterday on a freshly provisioned Ubuntu 17.10 box.

Thanks again for bringing this to our attention.